apps/src/code-studio/headerShare.js

```diff
@@ -45,6 +45,7 @@ export function shareProject(shareUrl) {
       <Provider store={getStore()}>
         <ShareDialog
           isProjectLevel={!!dashboard.project.isProjectLevel()}
+          isDance={dashboard.project.getStandaloneApp() === 'dance'}
```

Comment:
We already have this as a local variable called `appType`. I would suggest we just reuse that.

My next feedback is that it would be nice to have this prop be not be app-specific. Is there some more general concept that encompasses why dance gets shared and other apps do not? If nothing else, we could maybe call this something like `allowSignedOutShare`?